src/olympia/bandwagon/tests/test_views.py

```diff
@@ -1079,8 +1114,11 @@ def test_admin(self):
         self.grant_permission(self.user, 'Collections:Edit')
```

Comment:
Should this permission still be there?  Looks like `COLLECTIONS_EDIT` is unused entirely now.

Reply:
There might be other references to `'Collections:Edit'` - this one was just peeping out from the diff 😉

Reply:
I left it but modified the expected status code to 403 - So it tests that having it does not grant you the permission until later in the test `Admin:Curation` is added.